pkg/printers/internalversion/printers.go

```diff
@@ -99,17 +98,17 @@ func AddHandlers(h printers.PrintHandler) {
 		{Name: "Nominated Node", Type: "string", Priority: 1, Description: apiv1.PodStatus{}.SwaggerDoc()["nominatedNodeName"]},
 		{Name: "Readiness Gates", Type: "string", Priority: 1, Description: apiv1.PodSpec{}.SwaggerDoc()["readinessGates"]},
 	}
-	h.TableHandler(podColumnDefinitions, printPodList)
-	h.TableHandler(podColumnDefinitions, printPod)
+	_ = h.TableHandler(podColumnDefinitions, printPodList)
```

Comment:
Can you add a comment before the first time this is done explaining why it is OK to ignore these errors?

Reply:
Sure